packages/icons-angular/tasks/build.js

```diff
@@ -5,9 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const build = require('../src/build');
-const { reporter } = require('@carbon/cli-reporter');
+/**
+ * icons-angular disabled until memory issue resolved (circleci failing)
+ */
+// const build = require('../src/build');
```

Comment:
@lovemecomputer Are you removing these commented-out code, or is it for just testing CI?

Reply:
@asudoh working off of @joshblack 's suggestion, the intent is for this to be commented out in this PR, as the memory issues cause circleci to fail. after verifying that this is a fix (seeing all tests pass on this PR), i will create a new issue to fix the memory issue in `icons-angular` and link to this PR as reference :)

Reply:
OK thanks @lovemecomputer for explaining! For now, it'd be great if the commented out code here refers to the issue you create.

Reply:
@asudoh okay, will do!

Reply:
@asudoh link put in comment!